avm/ptn/azd/acr-container-app/modules/registry-access.bicep

```diff
@@ -0,0 +1,28 @@
+metadata name = 'ACR Pull permissions'
+metadata description = 'Assigns ACR Pull permissions to access an Azure Container Registry.'
+metadata owner = 'Azure/module-maintainers'
+
+@description('Required. The name of the container registry.')
+param containerRegistryName string
+
+@description('Required. The principal ID of the principal (user/group/identity) to assign the role to.')
+param principalId string
+
+var acrPullRole = subscriptionResourceId(
+  'Microsoft.Authorization/roleDefinitions',
+  '7f951dda-4ed3-4680-a7ca-43fe172d538d'
+)
+
+module aksAcrPull 'br/public:avm/ptn/authorization/resource-role-assignment:0.1.1' = {
+  name: 'acrpullrole-deployment'
+  params: {
```

Comment:
Misses passthru of `enableTelemetry` parameter

Reply:
See other comment

Reply:
@AlexanderSehr I have updated by your comments.